sdk/batch/batch/rollup.config.js

```diff
@@ -1,15 +1,12 @@
 import rollup from "rollup";
```

Comment:
I do not think this will work correctly for tests to run in the browser. Instead, use https://raw.githubusercontent.com/Azure/autorest.typescript/main/test/integration/generated/additionalProperties/rollup.config.js

Reply:
And you will need the rollup-related deps listed in https://github.com/Azure/autorest.typescript/blob/65c5aaa3ecf9ac199bbd331cc8892171034b0d88/test/integration/generated/additionalProperties/package.json#L18

Reply:
The rollup was not updated yet I think.

Reply:
I reverted it so there is no change to the file in this PR: https://github.com/gingi/azure-sdk-for-js/blob/%40batch/2021-06-01/sdk/batch/batch/rollup.config.js